drivers/hwmon/ina3221.c

```diff
@@ -204,8 +204,8 @@ static inline u32 ina3221_reg_to_interval_us(u16 config)
 	u32 channels = hweight16(config & INA3221_CONFIG_CHs_EN_MASK);
 	u32 vbus_ct_idx = INA3221_CONFIG_VBUS_CT(config);
 	u32 vsh_ct_idx = INA3221_CONFIG_VSH_CT(config);
-	u32 vbus_ct = ina3221_conv_time[vbus_ct_idx];
-	u32 vsh_ct = ina3221_conv_time[vsh_ct_idx];
+	int vbus_ct = ina3221_conv_time[vbus_ct_idx];
+	int vsh_ct = ina3221_conv_time[vsh_ct_idx];
```

Comment:
not sure if these should be changed;
i guess it's fine to keep; let's see what upstream says;
int/u32 convert a bit more seemlessly in C (vs C++); there is a lesser chance of bugs (than with pointer type conversions);